react/features/overlay/components/ReloadTimer.js

```diff
@@ -0,0 +1,141 @@
+         *
+         * @public
+         * @type {number}
+         */
+        interval: React.PropTypes.number,
+
+        /**
+         * The function that will be executed when timer finish (when
+         * this.state.current === this.props.end)
+         */
+        onFinish: React.PropTypes.func,
+
+        /**
+         * The start of the timer. The initial value for this.state.current.
+         *
+         * @public
+         * @type {number}
+         */
+        start: React.PropTypes.number,
+
+        /**
+         * The value which will be added to this.state.current on every step.
+         *
+         * @public
+         * @type {number}
+         */
+        step: React.PropTypes.number
+    }
+
+    /**
+     * Initializes a new ReloadTimer instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     * @public
+     */
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            current: this.props.start,
+            time: Math.abs(this.props.end - this.props.start)
+        };
+    }
+
+    /**
+     * React Component method that executes once component is mounted.
+     *
+     * @inheritdoc
+     * @returns {void}
+     * @protected
+     */
+    componentDidMount() {
+        AJS.progressBars.update('#reloadProgressBar', 0);
+
+        const intervalId
+            = setInterval(
+                    () => {
+                        if (this.state.current === this.props.end) {
+                            clearInterval(intervalId);
+                            this.props.onFinish();
+                        } else {
+                            this.setState((prevState, props) => {
+                                return {
+                                    current: prevState.current + props.step
+                                };
+                            });
+                        }
+                    },
+                    Math.abs(this.props.interval) * 1000);
+    }
+
+    /**
+     * React Component method that executes once component is updated.
+     *
+     * @inheritdoc
+     * @returns {void}
+     * @protected
+     */
+    componentDidUpdate() {
+        AJS.progressBars.update(
+                '#reloadProgressBar',
+                Math.abs(this.state.current - this.props.start)
+                    / this.state.time);
+    }
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement|null}
+     * @public
+     */
+    render() {
+        return (
+            <div>
+                <div
+                    className = 'aui-progress-indicator'
+                    id = 'reloadProgressBar'>
+                    <span className = 'aui-progress-indicator-value' />
+                </div>
+                <span
+                    className = 'reload_overlay_text'
+                    id = 'reloadSeconds'>
```

Comment:
Probably I was using it at some point and later forgot to remove it. I think it's safe to be removed now.